db-services/user-service/src/main/resources/log4j2.xml

```diff
@@ -3,7 +3,7 @@
                packages="org.apache.logging.log4j.test">
     <Appenders>
         <Appender type="Console" name="Main">
-            <Layout type="PatternLayout" pattern="%d{HH:mm:ss} %C.%M -- %m%n%ex%n"/>
+            <Layout type="PatternLayout" pattern="[user-service] %d{HH:mm:ss} %C.%M -- %m%n%ex%n"/>
```

Comment:
I think we should reflect the affiliation to the particular service in package hierarchy. Like `com.qreal.robots.userservice.server`.